src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts

```diff
@@ -8,61 +13,100 @@ import { FormControl, FormGroup, Validators } from '@angular/forms';
 })
 export class UbsClientProfilePageComponent implements OnInit {
   userForm: FormGroup;
+  public googleIcon = SignInIcons.picGoogle;
```

Comment:
there is no need to write public, since it's public by defafult

Reply:
fixed